### DISCUSSION TITLE
any way to make it work better in windowed mode?

### DISCUSSION THREAD

**Discussion by yepyep12** [tanjeffreyz/auto-maple#61] [Q&A]
since the game cant be windowed fullscreen its super annoying to switch to any other task for a second so im wondering if anything in the code can be slightly changed to make up for the minimap location

**Comment by tanjeffreyz** [comment]
Hi, do you play on 1366x768 or 1920x1080? The bot is currently set to capture 1366x768 [here](https://github.com/tanjeffreyz/auto-maple/blob/cb23be6308afc6dee149096df3b473b38a774735/config.py#L7), which should also work for 1920x1080. For other windowed resolutions, I'll see if I can find a fix. Thanks for bringing this up!

**Reply by yepyep12** [reply]
ill check again soon, i just saw that the cf1 map was a bit off when i tried a few days ago

**Reply by yepyep12** [reply]
It looks like this unless i manage to shove the top of the maplestory window off the top of my monitor

![image](https://user-images.githubusercontent.com/104903486/167536661-476db013-5584-4085-95e9-bda4236cb2cb.png)
